### ISSUE TITLE
"space-in-parens" false positive with backtick ` strings

### ISSUE THREAD

**Issue by feross** [eslint/eslint#3182]
With this code:

```js
var test = `(hi )`
console.log(test)
```

And this config:

```json
"space-in-parens": [2, "never"]
```

The following false positive occurs:

```
t.js:1:16: There should be no spaces inside this paren.
```

**Comment by eslintbot** [comment]
Thanks for the issue! We get a lot of issues, so this message is automatically posted to each one to help you check that you've included all of the information we need to help you.

Reporting a bug? Please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
1. The source code that caused the problem
1. The configuration you're using (for the rule or your entire config file)
1. The actual ESLint output complete with line numbers

Requesting a new rule? Please be sure to include:

1. The use case for the rule - what is it trying to prevent or flag?
1. Whether the rule is trying to prevent an error or is purely stylistic
1. Why you believe this rule is generic enough to be included

Requesting a feature? Please be sure to include:

1. The problem you want to solve (don't mention the solution)
1. Your take on the correct solution to problem

Including this information in your issue helps us to triage it and get you a response as quickly as possible.

Thanks!

**Comment by nzakas** [comment]
What version are you using? I think we might have fixed this.

**Comment by feross** [comment]
1.0.0-rc3

**Comment by nzakas** [comment]
Damn, guess not. Okay, we'll get this fixed.

**Comment by IanVS** [comment]
Working on it.
